doc/conf.py

```diff
@@ -131,7 +131,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ["_build", "data/**"]
+exclude_patterns = ["_build", "data/**", "whatsnew/fragments"]
```

Comment:
I don't know `towncrier`, but does it make sense to store fragments in separate directories for different versions?

Reply:
This is not necessary - once you rown `towncrier build`, the existing newsfragments are deleted. Thus you only keep the fragments of the current, upcoming release.

Reply:
Where are the old fragments stored ? Do we only store the generated result  ?

Reply:
Exactly. `towncrier build` appends to the specified output file, and deletes the existing news fragments.